src/index.html

```diff
@@ -2,10 +2,12 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
   <title>React App</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i&amp;subset=cyrillic,cyrillic-ext,latin-ext" rel="stylesheet">
```

Comment:
If you use google api OpenSans then you don't need to store it locally

Reply:
Sure. Forgot about those fonts :) Was going to use them but I will stay with Google fonts